platform/solr/solr-banana/src/main/resources/banana-int/Banana-Default-Dashboard.json

```diff
@@ -0,0 +1,550 @@
+{
+  "title": "Metadata Default Dashboard",
```

Comment:
Come to think of it, the current data displayed on the default dashboard is metacard modifications, creation dates, and metacard titles. I think the end idea I discussed with @brendan-hofmann and @pklinef would be to show validation errors / warnings but to leave that to the end user to configure and display.

So I think the validation warning / error message that I put in the default message might be a bit misleading.